src/solver/impls/slepc/slepc.cxx

```diff
@@ -336,437 +336,428 @@ void SlepcSolver::vecToFields(Vec &inVec){
       2. ?
    */
 
-  //Get pointer to data
-  PetscScalar *point;
-  VecGetArray(inVec,&point);
+  // Get pointer to data
+  const BoutReal* point;
+  VecGetArrayRead(inVec, &point);
```

Comment:
Is this backwards compatible?

Reply:
I'm pretty sure, yes -- we do the same thing in the PETSc solver